payments/stripe/forms.py

```diff
@@ -20,29 +22,53 @@ def __init__(self, *args, **kwargs):
             self.payment.change_status('rejected')
             raise RedirectNeeded(self.payment.get_failure_url())
 
+    def _handle_potentially_fraudulent_charge(self, charge, commit=True):
+        fraud_details = charge['fraud_details']
+        if fraud_details.get('stripe_report', None) == 'fraudulent':
+            reject_fraud_choice = FRAUD_CHOICES[2][0]
```

Comment:
I'm not sure this increases the readability when compared to `status = 'reject'`. There is a separate ticket to replace choices with proper enums.